src/components/portal/portal.test.tsx

```diff
@@ -33,13 +33,49 @@ import { mount } from 'enzyme';
 import { OuiPortal } from './portal';
 
 describe('OuiPortal', () => {
-  test('is rendered', () => {
-    const component = mount(
-      <div>
-        <OuiPortal>Content</OuiPortal>
-      </div>
-    );
+  afterEach(() => {
+    document.body.innerHTML = '';
+  });
+
+  it('should render OuiPortal', () => {
+    const component = mount(<OuiPortal>Content</OuiPortal>);
 
     expect(component).toMatchSnapshot();
   });
+
+  it('should attach Content to body', () => {
+    mount(<OuiPortal>Content</OuiPortal>);
+
+    expect(document.body.innerHTML).toEqual('<div>Content</div>');
+  });
```

Comment:
I see the content of these two tests are the same, can they be combined into one test or does it make more sense to keep them separated?

Reply:
Good idea, combined test cases